collectors/ebpf.plugin/ebpf.c

```diff
@@ -642,28 +651,36 @@ static inline void ebpf_set_thread_mode(netdata_run_mode_t lmode)
 /**
  * Enable specific charts selected by user.
  *
- * @param em      the structure that will be changed
- * @param enable the status about the apps charts.
+ * @param em             the structure that will be changed
+ * @param disable_apps   the status about the apps charts.
+ * @param disable_cgroup the status about the apps charts.
```

Comment:
`apps` -> `cgroups`? (or was this intentional)

Reply:
it was not, last commit fixed this.